app/src/main/java/com/bogdan/codeforceswatcher/util/CrashLogger.kt

```diff
@@ -0,0 +1,16 @@
+package com.bogdan.codeforceswatcher.util
+
+import android.os.Bundle
+import com.bogdan.codeforceswatcher.CwApp
+import com.google.firebase.analytics.FirebaseAnalytics
+
+object CrashLogger {
+
+    private val instance = FirebaseAnalytics.getInstance(CwApp.app)
+
+    fun logThrowable(t: Throwable) {
```

Comment:
just `log` would be good enough. And also you are logging an event, while there is a `logException` method available.